TMessagesProj/src/main/res/values/cg_strings.xml

```diff
@@ -204,7 +204,7 @@
     <string name="CG_Kangs_Delete">Delete CG stickerpack</string>
     <string name="CG_Kangs_Delete_Desc">This will delete EVERY added sticker!</string>
 
-    <string name="CG_VoiceRecognise">Recognise</string>
+    <string name="CG_VoiceRecognise">Recognize</string>
```

Comment:
Is there such a difference in how to write this word? *(In the British or American version)*

Reply:
Well 's' and 'z' are interchangeable and correspond to en_gb and en_us respectively.